code/_onclick/hud/human.dm

```diff
@@ -44,8 +44,7 @@
 
 /mob/living/carbon/human/proc/remake_hud() //used for preference changes mid-round; can't change hud icons without remaking the hud.
 	if(hud_used)
-		qdel(hud_used)
-		hud_used = null
+		QDEL_NULL(hud_used)
```

Comment:
You don't need the `if(hud_used)` check, because `QDEL_NULL` define already check for that.